Userland/Libraries/LibIPC/Encoder.cpp

```diff
@@ -159,6 +160,12 @@ Encoder& Encoder::operator<<(ByteBuffer const& value)
     return *this;
 }
 
+Encoder& Encoder::operator<<(JsonValue const& value)
+{
+    *this << value.to_string();
```

Comment:
This is dangerous as it creates broken serialized JSON for string `JsonValues` (check the `to_string()` implementation. Needs to call `serialize()` directly.

Reply:
:foot: :gun: